src/Ddeboer/Imap/Message.php

```diff
@@ -171,8 +171,11 @@ public function getHeaders()
             // \imap_header is much faster than \imap_fetchheader
             // \imap_header returns only a subset of all mail headers,
             // but it does include the message flags.
-            $headers = \imap_header($this->stream, $this->messageNumber);
-            $this->headers = new Message\Headers($headers);
+            try {
+                $headers = \imap_header($this->stream, $this->messageNumber);
+            } catch (\Exception $e) {
```

Comment:
I am no fan of try/catch but when I added it there was assuredly at least one exception...I think;)